detectable/src/main/java/com/synopsys/integration/detectable/detectables/gradle/inspection/inspector/GradleInspectorProxyInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.synopsys.integration.detectable.detectables.gradle.inspection.inspector;
+
+public class GradleInspectorProxyInfo {
+    private final String proxyHost;
+    private final String proxyPort;
+    private final String nonProxyHosts;
+
+    public GradleInspectorProxyInfo(final String proxyHost, final String proxyPort, final String nonProxyHosts) {
+        this.proxyHost = proxyHost;
+        this.proxyPort = proxyPort;
+        this.nonProxyHosts = nonProxyHosts;
+    }
+
+    public String getProxyHost() {
+        return proxyHost;
+    }
+
+    public String getProxyPort() {
+        return proxyPort;
+    }
+
+    public String getNonProxyHosts() {
+        return nonProxyHosts;
+    }
+}
```

Comment:
We already have a class for storing proxy related information. It's called ProxyInfo

Reply:
Ya I was going to just use that class but it doesn't have a field for nonProxyHosts.  I assumed that field was important based on its presence in the Java Networking API, but if it's not, then I agree we should just use ProxyInfo.  Or I guess I could talk to Eric about editing integration-rest...